crates/host/Cargo.toml

```diff
@@ -45,9 +45,9 @@ uuid = { workspace = true, features = ["serde"] }
 wascap = { workspace = true }
 wasmcloud-compat = { workspace = true }
 wasmcloud-control-interface = { workspace = true }
-wasmcloud-core = { workspace = true }
+wasmcloud-core = { workspace = true, features = ["otel"] }
 wasmcloud-runtime = { workspace = true }
-wasmcloud-tracing = { workspace = true }
+wasmcloud-tracing = { workspace = true, features = ["otel"] }
```

Comment:
I have enabled otel features for *wasmcloud-tracing* in *wasmcloud-host*, which causes otel features to be used whenever *wasmcloud-host* is used, such as wasmcloud_tracing::configure_tracing.

I've tried other fixes but none of them seem to be very well, such as having *wasmcloud-tracing* always import otel related dependencies.
Please let me know if you have a better suggestion!